test/built-ins/TypedArray/prototype/at/coerced-index-resize.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.at
+description: >
+  TypedArray.p.at behaves correctly on TypedArrays backed by resizable buffers
+  when the TypedArray is resized during parameter conversion
+includes: [resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+function TypedArrayAtHelper(ta, index) {
+  const result = ta.at(index);
+  return Convert(result);
```

Comment:
Same as earlier, the conversion done by `Convert()` isn't necessary in this test.

Reply:
Done.